debian/autobake-deb.sh

```diff
@@ -148,17 +62,13 @@ fakeroot dpkg-buildpackage -us -uc -I $BUILDPACKAGE_FLAGS
 # If the step above fails due to missing dependencies, you can manually run
 #   sudo mk-build-deps debian/control -r -i
 
-# Don't log package contents on Travis-CI to save time and log size
-if [[ ! $TRAVIS ]]
-then
-  echo "List package contents ..."
-  cd ..
-  for package in `ls *.deb`
-  do
-    echo $package | cut -d '_' -f 1
-    dpkg-deb -c $package | awk '{print $1 " " $2 " " $6}' | sort -k 3
-    echo "------------------------------------------------"
-  done
-fi
+echo "List package contents ..."
+cd ..
+for package in `ls *.deb`
+do
+  echo $package | cut -d '_' -f 1
+  dpkg-deb -c $package | awk '{print $1 " " $2 " " $6}' | sort -k 3
+  echo "------------------------------------------------"
+done
 
```

Comment:
Is this necessary as it seems to be just waste of log space to-do it every time? Should there be something like DEBUG_AUTOBAKE environmental variable to only print when wanted.